pkgs/misc/emulators/wine/base.nix

```
@@ -54,6 +54,8 @@ stdenv.mkDerivation ((lib.optionalAttrs (buildScript != null) {
   ++ lib.optionals xmlSupport    [ pkgs.libxml2 pkgs.libxslt ]
   ++ lib.optionals tlsSupport    [ pkgs.openssl pkgs.gnutls ]
   ++ lib.optionals openglSupport [ pkgs.libGLU_combined pkgs.mesa.osmesa pkgs.libdrm ]
+  ++ lib.optionals faudioSupport [ pkgs.faudio ]
+  ++ lib.optional vkd3dSupport [ pkgs.vkd3d ]
```

```suggestion
  ++ lib.optionals vkd3dSupport [ pkgs.vkd3d ]
```

That fix compilation for me